### PR TITLE
HPCC-9691 DOCS:Handling removal thor nodes

### PR DESCRIPTION
**Pull request by g-pan**
FIX HPCC-9691 DOCS:Handling removal thor nodes
Added section Redefining nodes in a Thor Cluster
(Ch. 6 Sect. 6) To Systems Admin doc.

Signed-off-by: G Panagiotatos greg.panagiotatos@lexisnexis.com

@jakesmith @JamesDeFabia please review

### CODE REVIEW

File: docs/HPCCSystemAdmin/HPCCSystemAdministratorsGuide.xml [hunk 1]
```diff
@@ -1195,7 +1195,50 @@ lock=/var/lock/HPCCSystems</programlisting>
...
+      to restructure the group. Dali will not automatically restructure an
+      existing group.</para>
+
+      <para>This is because existing published files reference the previous
+      cluster group state by name and therefore changing its structure would
+      invalidate those files and make the physical files inaccessible.</para>
+
+      <para>There are a couple of scenarios: <orderedlist>
+          <listitem>
+            <para>Replacing faulty node(s) If data files are replicated,
```
Comment (JamesDeFabia) [inline docs/HPCCSystemAdmin/HPCCSystemAdministratorsGuide.xml]: needs some punctuation

File: docs/HPCCSystemAdmin/HPCCSystemAdministratorsGuide.xml [hunk 2]
```diff
@@ -1195,7 +1195,50 @@ lock=/var/lock/HPCCSystems</programlisting>
...
+            command:</para>
+
+            <para><programlisting>updtdalienv &lt;environment_file&gt; -f</programlisting>
+            In cases where there is no replication, data loss may be
+            unavoidable and forcing the new group may still be the best
+            option.</para>
+          </listitem>
+
+          <listitem>
+            <para>Resizing the cluster If you are adding or removing Thor
```
Comment (JamesDeFabia) [inline docs/HPCCSystemAdmin/HPCCSystemAdministratorsGuide.xml]: needs some punctuation

File: docs/HPCCSystemAdmin/HPCCSystemAdministratorsGuide.xml [hunk 3]
```diff
@@ -1195,7 +1195,50 @@ lock=/var/lock/HPCCSystems</programlisting>
...
+            environment and accessible</emphasis>, you can <emphasis
+            role="bold">rename</emphasis> the group that is associated with
+            the Thor cluster (or the Cluster name if there is no group
+            name).</para>
+
+            <para>This will ensure all previously existing files, continue to
+            use the old group structure, while new files use the new group
+            structure.</para>
+          </listitem>
+        </orderedlist>In summary, if the environment changes it must be
```
Comment (JamesDeFabia) [inline docs/HPCCSystemAdmin/HPCCSystemAdministratorsGuide.xml]: In summary, if the Thor cluster changes it must be updated in the Dali.

**Comment by JamesDeFabia** [comment]
Looks good to me now, but let @jakesmith have the final say

**Comment by jakesmith** [comment]
Looks good to me.

**Pull request merged**